<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Query</class>
 <widget class="QWidget" name="Query">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>查找</string>
  </property>
  <property name="windowIcon">
   <iconset resource="lib.qrc">
    <normaloff>:/lib/fdogicon.png</normaloff>:/lib/fdogicon.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">*{font-family:Microsoft YaHei;}
#Query{
background-color: rgb(193, 255, 183);
}</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>7</number>
   </property>
   <property name="topMargin">
    <number>7</number>
   </property>
   <property name="rightMargin">
    <number>7</number>
   </property>
   <property name="bottomMargin">
    <number>7</number>
   </property>
   <item row="0" column="0">
    <widget class="QWidget" name="widget_2" native="true">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2" stretch="1,0,10">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QWidget" name="widget" native="true">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(98, 94, 240);</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout" stretch="1">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QLabel" name="label">
             <property name="font">
              <font>
               <family>Microsoft YaHei</family>
               <pointsize>12</pointsize>
               <stylestrategy>PreferAntialias</stylestrategy>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">color: rgb(255, 255, 255);</string>
             </property>
             <property name="text">
              <string>查 找</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer">
             <property name="font">
              <font>
               <family>Microsoft YaHei</family>
               <stylestrategy>PreferAntialias</stylestrategy>
              </font>
             </property>
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QToolButton" name="toolButton_2">
             <property name="font">
              <font>
               <family>Microsoft YaHei</family>
               <stylestrategy>PreferAntialias</stylestrategy>
              </font>
             </property>
             <property name="text">
              <string>...</string>
             </property>
             <property name="icon">
              <iconset resource="lib.qrc">
               <normaloff>:/lib/zuixiaohua.png</normaloff>:/lib/zuixiaohua.png</iconset>
             </property>
             <property name="iconSize">
              <size>
               <width>20</width>
               <height>20</height>
              </size>
             </property>
             <property name="autoRaise">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QToolButton" name="toolButton_4">
             <property name="font">
              <font>
               <family>Microsoft YaHei</family>
               <stylestrategy>PreferAntialias</stylestrategy>
              </font>
             </property>
             <property name="text">
              <string>...</string>
             </property>
             <property name="icon">
              <iconset resource="lib.qrc">
               <normaloff>:/lib/guanbi.png</normaloff>:/lib/guanbi.png</iconset>
             </property>
             <property name="iconSize">
              <size>
               <width>20</width>
               <height>20</height>
              </size>
             </property>
             <property name="autoRaise">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget_3" native="true">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>151</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">#widget_3{
border-style:solid;
border-width:2px;
}
</string>
        </property>
        <widget class="QLineEdit" name="lineEdit">
         <property name="geometry">
          <rect>
           <x>100</x>
           <y>70</y>
           <width>600</width>
           <height>41</height>
          </rect>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>30</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>600</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Microsoft YaHei</family>
           <stylestrategy>PreferAntialias</stylestrategy>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">border-color: rgb(98, 94, 240);
border-style:solid;
border-width:1px;
border-radius:3px;
background-color: rgb(243, 243, 243);</string>
         </property>
         <property name="placeholderText">
          <string>请输入用户账号进行查询</string>
         </property>
        </widget>
        <widget class="QPushButton" name="pushButton">
         <property name="geometry">
          <rect>
           <x>760</x>
           <y>60</y>
           <width>141</width>
           <height>61</height>
          </rect>
         </property>
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>35</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Microsoft YaHei</family>
           <pointsize>14</pointsize>
           <stylestrategy>PreferAntialias</stylestrategy>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">border-width:5px;
background-color: rgb(191, 191, 249);
border-radius:5px;
min-width:100px;
min-height:35px;</string>
         </property>
         <property name="text">
          <string>查找</string>
         </property>
        </widget>
        <widget class="QPushButton" name="addFriend">
         <property name="geometry">
          <rect>
           <x>365</x>
           <y>10</y>
           <width>100</width>
           <height>50</height>
          </rect>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>50</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Microsoft YaHei</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">border-width:5px;
background-color: rgb(191, 191, 249);
border-radius:5px;
min-width:100px;
min-height:50px;</string>
         </property>
         <property name="text">
          <string>加好友</string>
         </property>
        </widget>
        <widget class="QPushButton" name="addGroup">
         <property name="geometry">
          <rect>
           <x>517</x>
           <y>10</y>
           <width>100</width>
           <height>50</height>
          </rect>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>50</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Microsoft YaHei</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">border-width:5px;
background-color: rgb(240, 239, 253);
border-radius:5px;
min-width:100px;
min-height:50px;
color: rgb(180, 177, 184);</string>
         </property>
         <property name="text">
          <string>加群</string>
         </property>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget_4" native="true">
        <widget class="QPushButton" name="pushButton_3">
         <property name="geometry">
          <rect>
           <x>850</x>
           <y>70</y>
           <width>101</width>
           <height>51</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <family>Microsoft YaHei</family>
           <stylestrategy>PreferAntialias</stylestrategy>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(240, 239, 253);
border-style:solid;
color: rgb(0, 0, 0);
border-radius:5px;
</string>
         </property>
         <property name="text">
          <string>+好友</string>
         </property>
        </widget>
        <widget class="QPushButton" name="pushButton_4">
         <property name="geometry">
          <rect>
           <x>45</x>
           <y>70</y>
           <width>101</width>
           <height>71</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <family>Microsoft YaHei</family>
           <stylestrategy>PreferAntialias</stylestrategy>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">border-radius:33px;</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="iconSize">
          <size>
           <width>66</width>
           <height>66</height>
          </size>
         </property>
        </widget>
        <widget class="QLabel" name="label_2">
         <property name="geometry">
          <rect>
           <x>160</x>
           <y>80</y>
           <width>661</width>
           <height>31</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <family>Microsoft YaHei</family>
           <stylestrategy>PreferAntialias</stylestrategy>
          </font>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
        <widget class="QLabel" name="label_3">
         <property name="geometry">
          <rect>
           <x>160</x>
           <y>120</y>
           <width>391</width>
           <height>21</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <family>Microsoft YaHei</family>
           <stylestrategy>PreferAntialias</stylestrategy>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(114, 114, 114);</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
        <widget class="QLabel" name="label_4">
         <property name="geometry">
          <rect>
           <x>40</x>
           <y>10</y>
           <width>201</width>
           <height>41</height>
          </rect>
         </property>
         <property name="text">
          <string>你可能认识的人：</string>
         </property>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="lib.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>toolButton_2</sender>
   <signal>clicked()</signal>
   <receiver>Query</receiver>
   <slot>showMinimized()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>673</x>
     <y>11</y>
    </hint>
    <hint type="destinationlabel">
     <x>357</x>
     <y>271</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
